lib/src/components/NxSubmitMask/NxSubmitMask.scss

```diff
@@ -30,17 +31,18 @@
 
 .nx-submit-mask__message {
   @include container-vertical;
+  @include semi-bold();
 
-  background-color: $nx-off-white;
-  border: 2px solid $nx-slate;
-  box-shadow: -4px 4px 0px rgba($nx-slate, 0.65);
+  background-color: #fff;
+  border-radius: $nx-border-radius;
   margin: 0;
-  padding: 22px 12px;
+  padding: $nx-spacing-l;
+  min-width: 124px;
+  max-width: 250px;
 }
 
 .nx-submit-mask--success {
   .nx-submit-mask__message {
-    border-color: $nx-dark-green;
-    color: $nx-dark-green;
+    color: #099843;
```

Comment:
I'm seeing #1a832b in zeplin

Reply:
> I'm seeing #1a832b in zeplin

I remember grabbing the green from Zeplin, but it's clearly wrong so it's fixed here: https://github.com/sonatype/sonatype-react-shared-components/pull/182/commits/a501adad683c58369ef36082dd474034170848da